src/settings/Settings.ts

```diff
@@ -288,6 +288,11 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         displayName: _td('Autoplay GIFs and videos'),
         default: false,
     },
+    "enableMarkdown": {
+        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
+        displayName: _td('Format messages with Markdown'),
```

Comment:
As discussed, this might be less ambiguous if rephrased to "Format your messages with Markdown".

Reply:
On it.